src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task {
+    
+    // New attribute
+    public String dueDate;
```

Comment:
I think you can use private/protected attributes instead of public for data encapsulation